### PR TITLE
Add missing methods to FileTest module

### PR DESCRIPTION
**Pull request by antryb**
According to #430 there are small changes to make it better

### CODE REVIEW

**Comment by grosser** [comment]
how did you know all these are missing ? ... can we automate that check in a test ?
can you hit the other added methods with a test too ? (just something simple to avoid typos)

**Comment by antryb** [comment]
I have found it during the usage in my own project. The good question is how should correct tests look like? The best option is to have some kind of tests that validate presence of all methods for specific version according to Ruby documentation

File: lib/fakefs/file_test.rb [hunk 1]
```diff
@@ -23,6 +27,22 @@ def readable?(file_name)
...
+    def sticky?(file_name)
+      File.sticky?(file_name)
+    end
+
+    def symlink?(file_name)
+      File.symlink?(file_name)
+    end
+
+    def world_readable?(file_name)
+      File.new(file_name).stat.world_readable?
```
Comment (antryb) [inline lib/fakefs/file_test.rb]: Forced to use stat due to absence of world_readable? method in File module

**Comment by grosser** [comment]
v0.20.0 🎉

**Comment by grosser** [comment]
good enough for now, make a PR if you find something better :)

**Pull request merged**